x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -285,23 +282,23 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
     cy.get(FILTER_GROUP_CHANGED_BANNER).should('be.visible');
   });
 
-  it('Changed banner should hide on saving changes', () => {
+  it('should hide Changed banner on saving changes', () => {
```

Comment:
line 289, I find it a tiny bit confusing that sometimes we `waitForPageFilters` in the test itself and sometimes it is nested within the `task` method. It is happening here in the `saveFilterGroupControls`. I'm not saying we need to remove it, I'm just throwing it out here that it feels a bit weird to me that we're have it in this method but not in many other ones... I haven't dug too deep into this case, so feel free to ignore my comment if it doesn't apply here!

Reply:
So `waitForPageFilters` is needed anywhere user chooses a new value in the page filters or they save page filters. Anything that renders the loading state of page filters, warrants this method.  I will double check the presence of this method. 

But let me know if above explanation makes sense.

Reply:
question on the one on line 262:
```
// high should be scuccessfully selected.
cy.get(OPTION_LIST_VALUES(1)).contains('high');
waitForPageFilters();

visitWithTimeRange(CASES_URL); // navigate away from alert page
```

is this one needed? I don't understand why we would have to wait for anything if the next task is to move to a different page...

Reply:
We wait, because if we do not, the localstorage and url are not updated sometimes and cypress is too fast to perform next operation.